packages/camera/camera_avfoundation/ios/Classes/CameraPlugin_Test.h

```diff
@@ -48,4 +48,8 @@
 - (void)createCameraOnSessionQueueWithCreateMethodCall:(FlutterMethodCall *)createMethodCall
                                                 result:(FLTThreadSafeFlutterResult *)result;
 
+/// Returns number value if provided and positive, or nil.
+/// Used to parse values like framerate and bitrate, that are positive by nature.
++ (NSNumber *)positiveOrNil:(id)number error:(NSError **)error;
```

Comment:
Why does this need to be exposed? This is an implementation detail of create message handling, so I would expect it to be testable via calling create with bad arguments.

Reply:
test now calls `createCameraOnSessionQueueWithCreateMethodCall` with bad or good arguments.